scripts/gitlab/build_and_test.sh

```diff
@@ -65,18 +65,18 @@ if [[ -z ${hostconfig} ]]
 then
     # If no host config file was provided, we assume it was generated.
     # This means we are looking of a unique one in project dir.
-    hostconfigs=( $( ls "${project_dir}/"hc-*.cmake ) )
```

Comment:
I liked the prefix because it help making the filename unique. (the script accept only one result).
I agree that this should be safe though. There is no reason for umpire to have a `.cmake` file in the project root dir.